ospf6d/ospf6_lsa.h

```diff
@@ -18,6 +18,7 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
  */
 
+#include "lib/json.h"
```

Comment:
How does ospf6_lsa.h need json.h?  Please include the lib/json.h where it is needed.